pkg/model/components/kubecontrollermanager.go

```diff
@@ -79,11 +79,7 @@ func (b *KubeControllerManagerOptionsBuilder) BuildOptions(o interface{}) error
 	kcm.ClusterName = b.ClusterName
 	switch kops.CloudProviderID(clusterSpec.CloudProvider) {
 	case kops.CloudProviderAWS:
-		if b.IsKubernetesGTE("1.24") {
-			kcm.CloudProvider = "external"
-		} else {
-			kcm.CloudProvider = "aws"
-		}
+		kcm.CloudProvider = "aws"
```

Comment:
Isn't external the only accepted value starting 1.24?

Reply:
Lower down it sets it to `external` if `ExternalCloudControllerManager` is non-`nil`